pkg/get/get.go

```diff
@@ -123,20 +134,34 @@ func (tool Tool) Head(uri string) (int, string, http.Header, error) {
 
 func (tool Tool) GetURL(os, arch, version string, quiet bool) (string, error) {
 
-	if len(version) == 0 &&
-		(len(tool.URLTemplate) == 0 || strings.Contains(tool.URLTemplate, "https://github.com/")) {
-		if !quiet {
-			log.Printf("Looking up version for %s", tool.Name)
-		}
+	if len(version) == 0 {
+		if len(tool.URLTemplate) == 0 || strings.Contains(tool.URLTemplate, "https://github.com/") {
+			if !quiet {
+				log.Printf("Looking up version for %s on GitHub", tool.Name)
+			}
 
-		v, err := FindGitHubRelease(tool.Owner, tool.Repo)
-		if err != nil {
-			return "", err
-		}
-		if !quiet {
-			log.Printf("Found: %s", v)
+			v, err := FindGitHubRelease(tool.Owner, tool.Repo)
+			if err != nil {
+				return "", err
+			}
+			if !quiet {
+				log.Printf("Found: %s", v)
+			}
+			version = v
+		} else if strings.Contains(tool.URLTemplate, "https://releases.hashicorp.com") {
```

Comment:
I think this could work, but I wonder if there's a cleaner way to express it? For instance, when we also may want to download the latest version of kubectl, which has a text file showing the latest version in text?

Reply:
Hey @alexellis , I raised several implementation suggestions on https://github.com/alexellis/arkade/issues/560 but I didn't get a response regarding an overall solution. Please let me know if you'd like me to resolve that.
Instead, on this PR I'm only suggesting to resolve HashiCorp versions that has a unique pattern.